IntegrationTests/Sources/IntegrationTestSupport/Helpers.swift

```diff
@@ -177,7 +184,8 @@ func _sh(
 
     environment.merge(env, uniquingKeysWith: { $1 })
 
-    let result = try Process.popen(arguments: arguments.map { $0.description }, environment: environment)
+    let result = try Process.popen(
+        arguments: arguments.map { $0.description }, environment: environment)
```

Comment:
Does this use SwiftFormat or swift-format? Per [`CONTRIBUTING.md`](https://github.com/swiftlang/swift-package-manager/blob/main/CONTRIBUTING.md#creating-pull-requests`) we're using SwiftFormat here, which IIUC would format it differently.
```suggestion
        arguments: arguments.map { $0.description }, environment: environment
    )
```

Reply:
Looks like the vscode swift extension picked swift-format.  I will re-run with SwiftFormat.